public/history.php

```diff
@@ -187,6 +189,15 @@ function resize() {
   }
 </script>
 
+<script>
+  function convertDate() {
+    var field = document.gotodateform.dateinput;
+    var timestamp = new Date(field.value).getTime() / 1000;
+    document.gotodateform.d.value = timestamp;
```

Comment:
```suggestion
    const { dateinput, d } = document.gotodateform;
    const timestamp = new Date(dateinput.value).getTime() / 1000;
    d.value = timestamp;
```

nit: Functionally equivalent, but slightly safer for changes later down the road thanks to immutability.